task/sql-tasks.js

```diff
@@ -280,7 +465,27 @@ async function task_1_21(db) {
  * @return {array}
  */
 async function task_1_22(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+            cust.CompanyName AS 'CompanyName',
+            prod.ProductName AS 'ProductName',
+            ordet.UnitPrice AS 'PricePerItem'
+        FROM Customers AS cust
+        INNER JOIN Orders AS ord ON ord.CustomerID = cust.CustomerID
+        INNER JOIN OrderDetails AS ordet ON ordet.OrderID = ord.OrderID
+        INNER JOIN Products AS prod ON prod.ProductID = ordet.ProductID
+        WHERE ordet.UnitPrice = (
+            SELECT

```

Comment:
try to group by this subquery customer id and join this subquery on unitprice and customer id